Tests/GPU/CuptiTest/Exec/CUDA/mykernel.H

```diff
@@ -0,0 +1,32 @@
+#ifndef MY_KERNEL_H_
+#define MY_KERNEL_H_
+
+#include <AMReX_FArrayBox.H>
+
+AMREX_GPU_DEVICE AMREX_FORCE_INLINE
+void init_mf (int i, int j, int k,
+               amrex::Array4<amrex::Real> const& mf)
+{
+  mf(i,j,k) = 0.;
+}
+
+using clock_value_t = long long;
+
```

Comment:
Use the AMReX Macros for device and global.

Reply:
Now using the AMReX macros.